systemd/systemd.go

```diff
@@ -129,6 +131,40 @@ func Available() error {
 	return err
 }
 
+// Version returns systemd version
+func Version() (int, error) {
+	out, err := systemctlCmd("--version")
+	if err != nil {
+		return 0, err
+	}
+
+	// systemd version outpus is two lines - actual version and a list
+	// of features, e.g:
+	// systemd 229
```

Comment:
I think this example should also include the form: `systemd 229 (229.1-4ubuntu1)` to show why this is done in this relatively complicated way

Reply:
Yes, this is part of prerequsite PR which I forgot to merge here, pushed now.